airly/_private.py

```diff
@@ -38,6 +40,8 @@ async def get(self, request_path):
                 raise AirlyError(response.status, await response.text())
 
             data = await response.json()
+            self.requests_per_day = response.headers.get("X-RateLimit-Limit-day")
```

Comment:
If these headers are provided for responses other than 200, it would be great to update the relevant fields as well (for example in case of `HTTP 429 - Too Many Requests` response).

Reply:
I made some tests. It seems that `X-RateLimit-Limit-day` and `X-RateLimit-Remaining-day` are available when HTTP status code is `200` and `429`. During testing I noticed that sometimes despite HTTP `200` the server doesn't return these headers so I added checking if they are available.